src/ai/backend/client/func/base.py

```diff
@@ -41,6 +46,38 @@ def api_function(meth):
     return meth
 
 
+def resolve_field(
```

Comment:
Since this function works on a list of fields, let's rename it to `resolve_fields`.

Reply:
Also please fix up test failures.